src/Sarif.FunctionalTests/SpecExamples/Comprehensive.sarif

```diff
@@ -74,42 +82,153 @@
           "formattedRuleMessage": {
             "formatId": "default",
             "arguments": [
-              "n"
+              "p"
```

Comment:
Sure, your suggestion of `ptr` below is fine.

---
In reply to: [72279756](https://github.com/Microsoft/sarif-sdk/pull/464#discussion_r72279756) [](ancestors = 72279756)